neat_bot.py

```diff
@@ -141,6 +147,21 @@ async def on_message(message):
         response = bookService.getAllAssignedBooks()
     elif "help" in cmd and "book" in cmd:
         response = bookService.bookHelp()
+    elif cmd.startswith(botName) and "help" in cmd:
+        response = bookService.bookHelp() + help() + gameService.gameHelp()
```

Comment:
This is some tech debt - this is just concatenating the service help commands. Eventually, we probably want to put these together in a way were we don't have to keep thinking about maintaining this.